src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Represents a task that users can input.
+ */
+public class Task {
+    String desc;
```

Comment:
Better to spell this variable and any references to this in your other classes in full, it makes it clear what this is supposed to contain! (Does it contain the description of the task, or does it describe an object that is descending, or...? :) )